docs/more-info/iOS.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "iOS"
+description: "Our iOS SDK allows you to receive Apple HealthKit data via webhooks"
+---
+
+## Initial Setup
+
+**Enable HealthKit:**
+
+First thing you need to do is enable HealthKit capabilities for your app.
+In Xcode, select the project, click on the `Signing & Capabilities` tab and add the HealthKit capability.
+Make sure to select `Background Delivery` under `Capabilities` to be able to
+receive data via webhooks.
```

Comment:
should we make this one line so the text can automatically break up based on browser width?